src/Heart/index.js

```diff
@@ -69,8 +71,6 @@ const Heart = ({ color, size, sizeUnit, width, height }) => (
     height={height}
   >
     <div />
```

Comment:
@tienpham94 Have a question about those 3 divs. I think only 1 div is enough for heart spinner. Were they intended?